src/components/application_manager/test/state_controller/state_controller_test.cc

```diff
@@ -190,6 +193,7 @@ class StateControllerImplTest : public ::testing::Test {
       mock_connection_handler__settings;
   transport_manager_test::MockTransportManager mock_transport_manager;
   connection_handler::ConnectionHandlerImpl* conn_handler;
+  application_manager::MockMessageHelper* message_helper_mock_;
```

Comment:
@LitvinenkoIra , add to ctor ` Mock::VerifyAndClearExpectations(&mock_message_helper_);`